bot/src/auctions/debt.ts

```diff
@@ -0,0 +1,49 @@
+import type { DebtAuctionActive } from 'auctions-core/src/types';
+import { fetchActiveDebtAuctions } from 'auctions-core/src/debt';
+import { notifyDebt } from '../notify';
+
+const THRESHOLD_FOR_NEW_AUCTIONS = 5 * 60 * 1000;
```

Comment:
I see we have this constant in three seperate files now. Do we maybe want to move this value to [`variables.ts`](https://github.com/sidestream-tech/unified-auctions-ui/blob/main/bot/src/variables.ts)?